specification/databricks/resource-manager/Microsoft.Databricks/stable/2024-05-01/databricks.json

```diff
@@ -0,0 +1,1991 @@
+        },
+        "enhancedSecurityMonitoring": {
+          "type": "object",
+          "description": "Status of Enhanced Security Monitoring feature.",
+          "$ref": "#/definitions/EnhancedSecurityMonitoringDefinition"
+        }
+      }
+    },
+    "AutomaticClusterUpdateDefinition": {
+      "type": "object",
+      "description": "Status of automated cluster updates feature.",
+      "properties": {
+        "value": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "AutomaticClusterUpdateValue"
+          }
+        }
+      }
+    },
+    "EnhancedSecurityMonitoringDefinition": {
+      "type": "object",
+      "description": "Status of Enhanced Security Monitoring feature.",
+      "properties": {
+        "value": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "EnhancedSecurityMonitoringValue"
+          }
+        }
+      }
+    },
+    "ComplianceSecurityProfileDefinition": {
+      "type": "object",
+      "description": "Status of Compliance Security Profile feature.",
+      "properties": {
+        "complianceStandards": {
+          "description": "Compliance standards associated with the workspace.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ComplianceStandardDefinition"
+          }
+        },
+        "value": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "ComplianceSecurityProfileValue"
+          }
+        }
+      }
+    },
+    "ComplianceStandardDefinition": {
+      "description": "Compliance standard that can be associated with a workspace.",
+      "type": "string",
+      "enum": [
+        "NONE",
+        "HIPAA",
+        "PCI_DSS"
+      ],
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "ComplianceStandard"
+      }
+    },
+    "WorkspaceCustomParameterType": {
+      "description": "Provisioning status of the workspace.",
+      "readOnly": true,
+      "enum": [
+        "Bool",
+        "Object",
+        "String"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "CustomParameterType",
+        "modelAsString": true
+      }
+    },
+    "WorkspaceNoPublicIPBooleanParameter": {
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/WorkspaceCustomParameterType",
+          "description": "The type of variable that this is"
+        },
+        "value": {
+          "type": "boolean",
+          "description": "The value which should be used for this field.",
+          "default": true
```

Comment:
`value` is required so it should not have a default value.

Reply:
@mikekistler this breaking change is already reviewed and approved. Please find the review request details here. 

https://teams.microsoft.com/l/message/19:meeting_MzlmYzczNDctZGUxOC00MDI2LWFlOGYtYzRkYWRlZjVkMzk1@thread.v2/1711386952623?context=%7B%22contextType%22%3A%22chat%22%7D

https://msazure.visualstudio.com/One/_workitems/edit/27298678

After the review is finished @TimLovellSmith has asked us to refactor some code and that removed the approval again.

Reply:
It is just wrong to have a default for a required field. Please fix.

Reply:
"WorkspaceNoPublicIPBooleanParameter": is not required it is optional, not sure why you are saying required. Can you confirm?